tests/conftest.py

```diff
@@ -93,7 +111,18 @@ def clean_repo(request, tmp_path_factory, mocker, base_repo: Repository):
 
     # remove all traces, and change working directory back to base path
     os.chdir(str(base_repo.root))
-    shutil.rmtree(tmp_path)
+    try:
+        shutil.rmtree(tmp_path)
+    except PermissionError:
```

Comment:
I think this might be fixed now. I'll remove it and we can rerun tests to see

Reply:
On windows 3.7 it still fails - I will put it back in with a TODO

Space was never an issue on Windows so I would say lets have it in so that we can start using the venvs and new gh actions.